transports/rabbitmq/delayed-delivery.md

```diff
@@ -50,10 +56,43 @@ end
 
 The delay levels are connected in this manner, from highest (27) to lowest (0). Each delay level's routing key's add wildcards as needed so that they are looking at the portion of the message's routing key that corresponds to its delay level.
 
+The following table illustrates dead letter exchange and bindings for the delay queues. It's important to note the wildcard rules for RabbitMQ binding expressions, where *word* describes a dot-delimited segment:
+
+* `*` substitutes for exactly one word
+* `#` substitutes for 0 or more words
+
+| Exchange & Queue Name | x-dead-letter-exchange | Queue Binding                                               |
```

Comment:
This table still doesn't make it clear that there are both 0 and 1 bindings on the exchange.